### ISSUE TITLE
Code assist for `type` keywords in request or response

### ISSUE THREAD

**Issue by tedepstein** [RepreZen/KaiZen-OpenAPI-Editor#47]
The `type` key shows available values for parameters, but not for responses.  I just got an alert tone when I tried code assist at the end of this code block:

```
swagger: '2.0'
info:
  title: Uber API
  description: Move your app forward with the Uber API
  version: 1.0.0
host: api.uber.com
schemes:
  - https
basePath: /v1
produces:
  - application/json
paths:
  /products:
    get:
      summary: Product Types
      description: |
        The Products endpoint returns information about the *Uber* products
        offered at a given location. The response includes the display name
        and other details about each product, and lists the products in the
        proper display order.
      parameters:
        - name: latitude
          in: query
          description: Latitude component of location.
          required: true
          type: number
          format: double
        - name: longitude
          in: query
          description: Longitude component of location.
          required: true
          type: 
```

**Comment by tfesenko** [comment]
Can't reproduce. Possibly fixed in PR #34

**Comment by ghillairet** [comment]
Closing, this works in master.
